src/server/pfs/server/testing/server_test.go

```diff
@@ -7614,3 +7614,39 @@ func TestNilBranchNameStream(t *testing.T) {
 		return nil
 	}))
 }
+
+func TestWalkFileBug(t *testing.T) {
```

Comment:
A few high level things:
- This test should be changed to something like `TestWalkFileInvalid` rather than something with `Bug` in the name. `Bug` doesn't really tell someone what is being tested here.
- This test could be simplified to just creating two files (i.e. /a and /a/a) that lead to a collision rather than four.
- The test should ensure that an error is returned by inspect and walk calls that trigger collisions.

Reply:
Thank you! I'll change these.

Reply:
I changed the first two bullet points. The inspect does not actually trigger a collision in the current way. Do we want to remove inspect from the test or do we want to change the way we are validating?